docs/vhost-user-blk-testing.md

```diff
@@ -69,14 +69,18 @@ make
 ```
 
 ### Set the SPDK environment
+
+Run SPDK vhost target with 2 CPU cores, i.e., core 0 and 1.
 ```bash
 sudo HUGEMEM=2048 scripts/setup.sh
 sudo ./build/bin/vhost -S /var/tmp -s 1024 -m 0x3 &
 ```
 
 ### Create 512M block device
+
+Create a 512M block device with 4096 bytes block size.
 ```bash
-sudo scripts/rpc.py bdev_malloc_create 512 512 -b Malloc0
+sudo scripts/rpc.py bdev_malloc_create 512 4096 -b Malloc0
```

Comment:
Not sure why I was tagged, I don't know much about SPDK. I have a naive question here: why do you change 512 to 4096 here? Does the original not work for you? If there is a reason behind this change, it'd better be written in the commit message.

Reply:
Make sure that user know the meanings, because we use two 512, and if some one do not fully know the orders for the parameters will be confused.  PS: I am previously the SPDK developer.

Reply:
Thanks for the explanation.